package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Book-a-Car",
+  "version": "1.0.0",
+  "description": "This is car booking applications",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Zubair Ali",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "dotenv": "^9.0.0",
+    "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
+    "pg": "^8.6.0",
+    "pg-hstore": "^2.3.3",
+    "sequelize": "^6.6.2"
+  },
+  "devDependencies": {
```

Comment:
Add prettier, eslint with airbnb, and husky on pre-commit to validate eslint/prettier.